proto/beacon/p2p/v1/types.proto

```diff
@@ -86,21 +86,32 @@ message AttestationDataAndCustodyBit {
 }
 
 message Validator {
-  bytes pubkey = 1;
-  // TODO(781): The usage of withdrawal_credentials is not defined in spec. Not used in Prysm yet.
-  bytes withdrawal_credentials_hash32 = 2;
-  uint64 activation_epoch = 5;
-  uint64 exit_epoch = 6;
-  uint64 withdrawal_epoch = 7;
-  uint64 slashed_epoch = 8;
+  // BLS public key
+  bytes pubkey = 1; // bytes48
+  // Withdrawal credentials
+  bytes withdrawal_credentials = 2; //bytes32
+  // Epoch when became eligible for activation
+  uint64 activation_eligibility_epoch = 3;
+  // Epoch when validator activated
+  uint64 activation_epoch = 4;
+  // Epoch when validator exited
+  uint64 exit_epoch = 5;
+  // Was the validator slashed
+  bool slashed = 6;
+  // Rounded balance
+  uint64 high_balance = 7;
+  // Deprecated
```

Comment:
```suggestion

  // Deprecated
  // TODO(2307): Remove deprecated fields
```

Reply:
Can you also reorganize the deprecated fields so that they are in the same group?